lib/git-hooks.js

```diff
@@ -111,6 +117,22 @@ module.exports = {
                 return path.resolve(hooksDirname, hookName);
             });
 
+            var ignoreList = [];
+            try {
```

Comment:
I believe that git-check-ignore use .gitignore from the home directory. So if we could just read .gitignore from project directory there won't be the case for git-check-ignore command